web-configs/nginx/nginx-no-ssl.conf

```diff
@@ -0,0 +1,59 @@
+# Define the location for where cobalt web runs. Change the port to fit your needs.
+upstream cobaltweb {
+    server 127.0.0.1:9000;
+    keepalive 64;
+}
+
+# Define the location for where cobalt API runs. Change the port to fit your needs.
+upstream cobaltapi {
+    server 127.0.0.1:9001;
+    keepalive 64;
+}
+
+# Server block definition for cobalt web.
+server {
+    # Listen for for requests to the domain on port 80.
+    listen 80;
+    listen [::]:80;
+    server_name co.wukko.sh;
+	
+    # Set loggging.
+    access_log /var/log/nginx/co.wukko.sh.access.log;
+    error_log /var/log/nginx/co.wukko.sh.error.log;
```

Comment:
i feel like instance owners should be encouraged to disable logging for privacy instead (say setting it to `/dev/null`), but thats just my $0.02